AvatarImage allows missing alt text

`src/components/ui/avatar.tsx` defines `AvatarImage` as a thin wrapper around `<img>`, but the wrapper currently does not guarantee an `alt` attribute.

That leaves multiple existing call sites able to render avatar images without alt text. The current lint output also flags the component with `jsx-a11y/alt-text`.

Repro on current `master`:

```bash
pnpm install --ignore-scripts --frozen-lockfile
pnpm exec eslint src/components/ui/avatar.tsx src/components/ui/avatar.test.tsx
```

Current result includes:

```text
src/components/ui/avatar.tsx
  23:3  warning  img elements must have an alt prop, either with meaningful text, or an empty string for decorative images  jsx-a11y/alt-text
```

Expected: `AvatarImage` should either preserve a caller-provided `alt`, or render a safe empty alt by default for decorative avatar images.

--- a/src/components/ui/avatar.test.tsx
+++ b/src/components/ui/avatar.test.tsx
@@ -1,0 +1,18 @@
+import { render, screen } from "@testing-library/react";
+import { describe, expect, it } from "vitest";
+
+import { AvatarImage } from "./avatar";
+
+describe("AvatarImage", () => {
+  it("uses an empty alt attribute by default", () => {
+    const { container } = render(<AvatarImage src="/avatar.png" />);
+
+    expect(container.querySelector("img")).toHaveAttribute("alt", "");
+  });
+
+  it("preserves an explicit alt attribute", () => {
+    render(<AvatarImage src="/avatar.png" alt="Ada Lovelace" />);
+
+    expect(screen.getByRole("img", { name: "Ada Lovelace" })).toBeInTheDocument();
+  });
+});

--- a/src/components/ui/avatar.tsx
+++ b/src/components/ui/avatar.tsx
@@ -1,46 +1,36 @@
 import * as React from "react";
 import { cn } from "@/lib/utils";
 
-const Avatar = React.forwardRef<
-  HTMLSpanElement,
-  React.HTMLAttributes<HTMLSpanElement>
->(({ className, ...props }, ref) => (
-  <span
-    ref={ref}
-    className={cn(
-      "relative flex h-10 w-10 shrink-0 overflow-hidden rounded-full",
-      className
-    )}
-    {...props}
-  />
-));
+const Avatar = React.forwardRef<HTMLSpanElement, React.HTMLAttributes<HTMLSpanElement>>(
+  ({ className, ...props }, ref) => (
+    <span
+      ref={ref}
+      className={cn("relative flex h-10 w-10 shrink-0 overflow-hidden rounded-full", className)}
+      {...props}
+    />
+  )
+);
 Avatar.displayName = "Avatar";
 
-const AvatarImage = React.forwardRef<
-  HTMLImageElement,
-  React.ImgHTMLAttributes<HTMLImageElement>
->(({ className, ...props }, ref) => (
-  <img
-    ref={ref}
-    className={cn("aspect-square h-full w-full", className)}
-    {...props}
-  />
-));
+const AvatarImage = React.forwardRef<HTMLImageElement, React.ImgHTMLAttributes<HTMLImageElement>>(
+  ({ className, alt = "", ...props }, ref) => (
+    <img ref={ref} alt={alt} className={cn("aspect-square h-full w-full", className)} {...props} />
+  )
+);
 AvatarImage.displayName = "AvatarImage";
 
-const AvatarFallback = React.forwardRef<
-  HTMLSpanElement,
-  React.HTMLAttributes<HTMLSpanElement>
->(({ className, ...props }, ref) => (
-  <span
-    ref={ref}
-    className={cn(
-      "flex h-full w-full items-center justify-center rounded-full bg-muted",
-      className
-    )}
-    {...props}
-  />
-));
+const AvatarFallback = React.forwardRef<HTMLSpanElement, React.HTMLAttributes<HTMLSpanElement>>(
+  ({ className, ...props }, ref) => (
+    <span
+      ref={ref}
+      className={cn(
+        "flex h-full w-full items-center justify-center rounded-full bg-muted",
+        className
+      )}
+      {...props}
+    />
+  )
+);
 AvatarFallback.displayName = "AvatarFallback";
 
 export { Avatar, AvatarImage, AvatarFallback };

